frontend/src/components/atoms/elements/Image.tsx

```diff
@@ -1,101 +1,85 @@
 import { useState } from 'react';
-
 import Skeleton from '@mui/material/Skeleton';
+import Lightbox from 'yet-another-react-lightbox';
+import 'yet-another-react-lightbox/styles.css';
+import Zoom from 'yet-another-react-lightbox/plugins/zoom';
+import Download from 'yet-another-react-lightbox/plugins/download';
 
-import { type IImageElement } from 'client-types/';
+import { type IImageElement, useConfig } from '@chainlit/react-client';
 
 import { FrameElement } from './Frame';
 
 interface Props {
   element: IImageElement;
 }
 
-const handleImageClick = (name: string, src: string) => {
-  const width = window.innerWidth / 2;
-  const height = window.innerHeight / 2;
-  const left = window.innerWidth / 4;
-  const top = window.innerHeight / 4;
-
-  const newWindow = window.open(
-    '',
-    '_blank',
-    `width=${width},height=${height},left=${left},top=${top}`
-  );
-  if (newWindow) {
-    newWindow.document.write(`
-      <html>
-        <head>
-          <title>${name}</title>
-          <link rel="icon" href="/favicon">
-          <style>
-            body {
-              margin: 0;
-              display: flex;
-              flex-direction: column;
-              justify-content: center;
-              align-items: center;
-              height: 100vh;
-              background-color: rgba(0, 0, 0, 0.8);
-            }
-            img {
-              max-width: 100%;
-              max-height: calc(100% - 50px);
-            }
-            a {
-              margin: 10px 0;
-              color: white;
-              text-decoration: none;
-              font-size: 15px;
-              background-color: rgba(255, 255, 255, 0.2);
-              padding: 8px 12px;
-              border-radius: 5px;
-            }
-            a:hover {
-              background-color: rgba(255, 255, 255, 0.4);
-            }
-          </style>
-        </head>
-        <body>
-          <img src="${src}" alt="${name}" />
-          <a href="${src}" download="${name}">Download</a>
-        </body>
-      </html>
-    `);
-    newWindow.document.close();
-  }
-};
-
 const ImageElement = ({ element }: Props) => {
   const [loading, setLoading] = useState(true);
+  const [lightboxOpen, setLightboxOpen] = useState(false);
+  const config = useConfig();
 
   if (!element.url) {
     return null;
   }
 
+  const enableLightbox = config.config?.features.image_lightbox && element.display === 'inline';
+
+  const handleImageClick = () => {
+    if (enableLightbox) {
+      setLightboxOpen(true);
+    }
+  };
+
+
```

Comment:
I like how this lightbox code is much more minimal. However, if I read this well, with the lightbox disabled _the old image click behaviour is entirely removed_.

That's not the way forward; developers and users expect what worked before to keep working towards the future and not suddenly disappear. So let's truly make lightbox optional and make sure that we _retain the exact legacy behaviour unless the lightbox is enabled_.

Reply:
I agree with this, but since the original feature was not released at the time of making this, i figured that it could be removed all together. Ill fix that.